### ISSUE TITLE
SpruceOS

### ISSUE THREAD

**Issue by Sundownersport** [link6155/pbmini-onion#1]
Awesome theme!

can you adapt it to work on spruce for the Miyoo A30?

it’s very similar to onion, this will actually sort of work already on spruce, there’s just a few new icons and assets that spruce needs.

lol I just realized you meant a new system icon….. not another device

**Comment by lnkd61** [comment]
Yeah I got some initial support for SpruceOS in place, like an icon for N64. The background is upside down though and I'm not sure if that can be fixed. Will try to see if I can pick up an A30 to do some testing.

**Comment by Sundownersport** [comment]
Awesome!

I am the team lead on spruce. DM me on discord and I can try to help you figure out what’s needed
